src/components/NcSettingsSelectGroup/NcSettingsSelectGroup.vue

```diff
@@ -131,8 +158,25 @@ export default {
 		},
 	},
 	methods: {
-		update() {
-			this.$emit('input', this.inputValue.map((element) => element.id))
+		update(updatedValue) {
+			const value = updatedValue.map((element) => element.id)
+			/** Emitted when the groups selection changes<br />**Payload:** `value` (`Array`) - *Ids of selected groups */
+			this.$emit('input', value)
+		},
+		/**
+		 * Load initial groups if not already loaded
+		 */
+		async initGroups() {
+			if (Object.keys(this.groups).length === 0) {
```

Comment:
When groups are selected they don't have their display name:
![Bildschirmfoto vom 2023-05-17 07-06-09](https://github.com/nextcloud/nextcloud-vue/assets/213943/66e6beae-c8dd-446d-915f-dc7cb4527b44)

As soon as I click into the field once, the displayname is loaded and used:
![Bildschirmfoto vom 2023-05-17 07-06-14](https://github.com/nextcloud/nextcloud-vue/assets/213943/6bcf8db5-2910-412a-a308-4069292fc62e)

you can easily create groups with diverging displaynames with occ:
```
occ group:add --display-name "Display name" `uuidgen`
```

Reply:
The "problem" is a search per selected group id might not be acceptable.
So I guess we should require that id+displayname of currently selected groups are given as a property.

Reply:
If you need an app for testing, you can use the announcementscenter with this branch:
https://github.com/nextcloud/announcementcenter/pull/648 
Setting is in the additional settings block

Reply:
@nickvergessen I fixed the problem of the missing display name as well as I add a debounce to the input so that it does not spam the API with requests while typing